.github/workflows/main.yml

```diff
@@ -0,0 +1,18 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: cachix/install-nix-action@v12
+      - uses: cachix/cachix-action@v8
+        with:
+          name: gi-gtk-declarative
+          authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
+      - name: 'Build and Test Libraries and Examples'
+        run: nix-build -j2 --arg doCheck true
+      - name: 'Build Docs'
+        run: nix-shell -j2 --run 'cd docs && mkdocs build'
```

Comment:
The travis build didn't build the docs, but I guess this is a handy thing to have.